src/components/attendee/InputSelect/InputSelect.tsx

```diff
@@ -46,30 +45,27 @@ export const InputSelect: React.FC<InputSelectProps> = ({
 }) => {
   const containerClassNames = classNames(
     styles.inputSelect,
-    {
-      [styles.invalid]: isDefined(error),
-    },
-    containerClassName
-  );
-  const selectContainerClassNames = classNames(
-    styles.inputSelect,
-    {
-      [styles.invalid]: isDefined(error),
-    },
     containerClassName
   );
 
+  const wrapperClassNames = classNames(styles.inputWrapper, {
+    [styles.invalid]: isDefined(error),
```

Comment:
[Q] does empty string for `error` mean the class `styles.invalid` should be added? If yes, a comment would be nice, and if not, I guess:

```suggestion
    [styles.invalid]: error,
```

Reply:
react-hook-form currently returns either `undefined` or an error object for this field, so an empty string wouldn't be the case
I've removed that `isDefined` anyway, just in case anything changes in that library
[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)